understanding/21/non-text-contrast.html

```diff
@@ -388,6 +388,16 @@ <h4>Infographics</h4>
 				<p>An infographic can use text which meets the other criteria to minimise the number of graphical objects required for understanding. For example, using text with sufficient contrast to provide the values in a chart. A long description would also be sufficient because then the infograph is not relied upon for understanding.</p>
 
 				</section>
+				
+				<section>
+					<h4>Symbolic text characters</h4>
+					<p>For the purposes of this SC, when text is used in a symbolic manner, rather than directly <q>expressing something in human language</q> as per the definition of <a>text</a> in WCAG, it is considered <em>non-text content</em>.</p>
```

Comment:
```suggestion
					<p>For the purposes of this Success Criterion, when text is used in a symbolic manner, rather than directly <q>expressing something in human language</q> as per the definition of <a>text</a> in WCAG, it is considered <em>non-text content</em>.</p>
```